### PR TITLE
pallet-para-config: Remove remnant WeightInfo functions

### PR DESCRIPTION
**Pull request by ggwpez**
https://github.com/paritytech/polkadot/pull/6271 removed the benchmark, so the weight functions must also go (not needed anymore).

### CODE REVIEW

**Comment by ggwpez** [comment]
bot bench $ runtime kusama-dev runtime_parachains::configuration

**Comment by ggwpez** [comment]
bot cancel  
bot clean  
bot bench $ runtime kusama-dev runtime_parachains::configuration

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2917154

File: runtime/kusama/src/weights/runtime_parachains_configuration.rs [hunk 1]
```diff
@@ -95,24 +95,6 @@ impl<T: frame_system::Config> runtime_parachains::configuration::WeightInfo for
...
-		//  Measured:  `90`
-		//  Estimated: `1575`
-		// Minimum execution time: 9_443_000 picoseconds.
-		Weight::from_parts(9_679_000, 0)
-			.saturating_add(Weight::from_parts(0, 1575))
-			.saturating_add(T::DbWeight::get().reads(3))
-			.saturating_add(T::DbWeight::get().writes(1))
-	}
-	/// Storage: Benchmark Override (r:0 w:0)
-	/// Proof Skipped: Benchmark Override (max_values: None, max_size: None, mode: Measured)
```
Comment (joepetrowski) [inline runtime/kusama/src/weights/runtime_parachains_configuration.rs:115]: Keep these two lines?

**Review by joepetrowski — APPROVED** [review]

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**